### PR TITLE
Added Portuguese transalations

### PR DESCRIPTION
**Pull request by Santos-Enoque**
## Description
I have added an .arb translation file for Portuguese translations
## Type of Change

- [ x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/l10n/arb/app_pt.arb [hunk 1]
```diff
@@ -0,0 +1,225 @@
+{
+  "@@locale": "pt",
+  "play": "Jogar",
+  "@play": {
+    "description": "Texto exibido no botão de jogar da página de inicial"
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:5]:
```suggestion
    "description": "Texto exibido no botão de jogar da página inicial"
```

File: lib/l10n/arb/app_pt.arb [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  "howToPlay": "Como Jogar",
+  "@howToPlay": {
+    "description": "Texto exibido no botão de como jogar da página de inicial"
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:9]:
```suggestion
    "description": "Texto exibido no botão de como jogar da página inicial"
```

File: lib/l10n/arb/app_pt.arb [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  "tipsForFlips": "Dicas para flips",
+  "@tipsForFlips": {
+    "description": "Texto exibido na página de inicial no botão de como jogar"
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:13]:
```suggestion
    "description": "Texto exibido na página inicial no botão de como jogar"
```

File: lib/l10n/arb/app_pt.arb [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  "launchControls": "Controles de Largada",
+  "@launchControls": {
+    "description": "Texto exibido na caixa de diálogo de como jogar com os controles de largada"
+  },
+  "flipperControls": "Controles do Flipper",
+  "@flipperControls": {
+    "description": "Texto exibido na caixa de diálogo de como jogar com os controles de largada"
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:21]:
```suggestion
    "description": "Texto exibido na caixa de diálogo de como jogar com os controles de flippers"
```

File: lib/l10n/arb/app_pt.arb [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  "to": "Para",
+  "@to": {
+    "description": "Texto exibido para a palavra para"
+  },
+  "launch": "LANÇAR",
+  "@launch": {
+    "description": "Texto exibido para a palavra lançar"
+  },
+  "tapLeftRightScreen": "Toque na Tela esquerda/direita",
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:35]:
```suggestion
  "tapLeftRightScreen": "Toque no lado esquerdo/direito da Tela",
```

File: lib/l10n/arb/app_pt.arb [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+  "@tapLeftRightScreen": {
+    "description": "Texto exibido em como fazer o flip no celular"
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:37]:
```suggestion
    "description": "Texto exibido em como usar os flippers no celular"
```

File: lib/l10n/arb/app_pt.arb [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+  "@flip": {
+    "description": "Texto exibido para a palavra FLIP"
+  },
+  "start": "Começar",
+  "@start": {
+    "description": "Texto exibido no botão de iniciar na página de seleção de personagem"
+  },
+  "select": "Selecionar",
+  "@select": {
+    "description": "Texto exibido no botão selecionar personagem na caixa de diálogo de seleção de personagens - "
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:49]:
```suggestion
    "description": "Texto exibido no botão selecionar personagem na caixa de diálogo de seleção de personagens"
```

File: lib/l10n/arb/app_pt.arb [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  "linkBoxFlutterLinkText": "Flutter",
+  "@linkBoxFlutterLinkText": {
+    "description": "Texto no link mostrado na caixa de link que navega para a página do Flutter"
+  },
+  "linkBoxFirebaseLinkText": "Firebase",
+  "@linkBoxFirebaseLinkText": {
+    "description": "Texto no link mostrado na caixa de link que leva à página do Firebase"
+  },
+  "linkBoxOpenSourceCode": "Código-fonte aberto",
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:123]:
```suggestion
  "linkBoxOpenSourceCode": "Código Aberto",
```

File: lib/l10n/arb/app_pt.arb [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+  "initialsErrorMessage": "Por favor, tente uma combinação diferente de letras",
+  "@initialsErrorMessage": {
+    "description": "Mensagem exibida quando o envio das iniciais falha"
+  },
+  "leaderboardErrorMessage": "Sem conexão. A funcionalidade de placar e compartilhamento não está disponível.",
+  "@leaderboardErrorMessage": {
+    "description": "Texto mostrado quando a tabela de classificação teve um erro ao carregar"
+  }
+,
+  "letEveryone": "Deixe todos saberem sobre o I/O Pinball",
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:204]:
```suggestion
  "letEveryone": "Mostre o I/O Pinball a todos",
```

File: lib/l10n/arb/app_pt.arb [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+    "description": "Texto exibido na tela de compartilhamento para descrição"
+  },
+  "bySharingYourScore": "compartilhando sua pontuação com sua preferida",
+  "@bySharingYourScore": {
+    "description": "Texto exibido na tela de compartilhamento para descrição"
+  },
+  "socialMediaAccount": "conta nas redes sociais!",
+  "@socialMediaAccount": {
+    "description": "Texto exibido na tela de compartilhamento para descrição"
+  },
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:215]:
```suggestion
  "bySharingYourScore": "compartilhando sua pontuação na sua",
  "@bySharingYourScore": {
    "description": "Texto exibido na tela de compartilhamento para descrição"
  },
  "socialMediaAccount": "rede social preferida!",
  "@socialMediaAccount": {
    "description": "Texto exibido na tela de compartilhamento para descrição"
  },
```

File: lib/l10n/arb/app_pt.arb [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+  "iGotScoreAtPinball": "Eu tenho {score} pontos no #IOPinball, você pode bater minha pontuação?\nNos vemos no #GoogleIO!",
```
Comment (thatfiredev) [inline lib/l10n/arb/app_pt.arb:216]:
```suggestion
  "iGotScoreAtPinball": "Consegui {score} pontos no #IOPinball, você pode bater minha pontuação?\nNos vemos no #GoogleIO!",
```

**Review by thatfiredev — COMMENTED** [review]
Thanks @Santos-Enoque !
Just left a few suggestions

File: lib/l10n/arb/app_pt.arb [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+  "@@locale": "pt",
+  "play": "Jogar",
+  "@play": {
+    "description": "Texto exibido no botão de jogar da página inicial"
+  },
+  "howToPlay": "Como Jogar",
+  "@howToPlay": {
+    "description": "Texto exibido no botão de como jogar da página inicial"
+  },
+  "tipsForFlips": "Dicas para flips",
```
Comment (pedromassango) [inline lib/l10n/arb/app_pt.arb:11]:
```suggestion
  "tipsForFlips": "Dicas para arremesso",
```

**Review by pedromassango — CHANGES_REQUESTED** [review]

File: lib/l10n/arb/app_pt.arb (hunk 12 above)
Comment (Santos-Enoque) [reply lib/l10n/arb/app_pt.arb:11]: I was searching for how flips are called in Portuguese! Thanks Pedro

**Review by pedromassango — APPROVED** [review]
LGTM

**Comment by tomarra** [comment]
Thanks for the contribution @Santos-Enoque! Would you be able to provide a short screen recording or screenshots of the translations running in the application? Some of these strings are shown in Flame instead of Flutter so the auto scaling isn't always 100%. Just want to make sure we don't have any overlapping or clipping.

File: lib/l10n/arb/app_pt.arb [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+  "@gotoIO": {
+    "description": "Texto para acessar o link do site de I/O na tela de informações."
+  },
+  "learnMore": "Saiba mais sobre como construir jogos no Flutter com",
+  "@learnMore": {
+    "description": "Texto mostrado na descrição da tela de informações"
+  },
+  "firebaseOr": "Firebase ou mergulhe direto no",
+  "@firebaseOr": {
+    "description": "Text shown on description of info screen"
```
Comment (cassioso) [inline lib/l10n/arb/app_pt.arb:181]: This line was overlooked

```suggestion
    "description": "Texto exibido na descrição da tela de informações"
```

**Review by cassioso — COMMENTED** [review]
nit: replace all 'mostrado' by 'exibido'

**Comment by Hixie** [comment]
@Santos-Enoque thanks for your contribution.
@tomarra can you take a look and decide what should be done with this PR?

File: lib/l10n/arb/app_pt.arb [hunk 14]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  "replay": "Repetir",
+  "@replay": {
+    "description": "Texto exibido na página de compartilhamento para botão de repetição"
+  },
+  "ioPinball": "I/O Pinball",
+  "@ioPinball": {
+    "description": "I/O Pinball - Nome do jogo"
+  },
+  "shareYourScore": "Partilhe sua pontuação",
```
Comment (erickzanardo) [inline lib/l10n/arb/app_pt.arb:159]:
```suggestion
  "shareYourScore": "Compartilhe sua pontuação",
```

**Comment by erickzanardo** [comment]
Left one small suggestions, other than that, LGTM.

CI needs to be solved though, spell checker is failing due to the addition of the Portuguese words, Ideally would be nice to add a pt spell checker for this file, but AFAIK the spell check action only supports us and gb english, so maybe we could just ignore the portugue arb file on the step.

File: lib/l10n/arb/app_pt.arb (hunk 14 above)
Comment (Santos-Enoque) [reply lib/l10n/arb/app_pt.arb:159]: Thank you very much Erick

**Comment by Santos-Enoque** [comment]
> Left one small suggestions, other than that, LGTM.
> 
> CI needs to be solved though, spell checker is failing due to the addition of the Portuguese words, Ideally would be nice to add a pt spell checker for this file, but AFAIK the spell check action only supports us and gb english, so maybe we could just ignore the portugue arb file on the step.

Is there something I should do to take care of the spell_check?

**Comment by Hixie** [comment]
Thanks for your contribution and sorry for making you jump through so many hoops given what I'm about to say... We've actually decided to archive this repo since we don't have anyone working on it full time these days, since it was mostly a demo for an event, but please don't hesitate to fork the repo (it's open source!) if you want to keep it alive!

**Pull request closed without merging**